### PR TITLE
HomeWork11

### PR DESCRIPTION
**Pull request by YeaaaaH**

### CODE REVIEW

File: HW/src/test/java/homework11/ArrayListTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        assertThat(listArray.get(1), is(user3));
+    }
+    @Test
+    public void addAllMethodTest() {
+        List<User> userList = new java.util.LinkedList<>();
+        userList.add(user1);
+        userList.add(user2);
+        userList.add(user3);
+        listArray.addAll(userList);
+        System.out.println(listArray.isEmpty());
```
Comment (Siril-O) [inline HW/src/test/java/homework11/ArrayListTest.java]: no printlines in test

File: HW/src/test/java/homework11/ArrayListTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+public class ArrayListTest {
+    ArrayList<User> listArray;
+    User user1;
+    User user2;
+    User user3;
+    User user4;
+
+    @Before
+    public void init() {
+        user1 = new User(1, "Ivan");
```
Comment (Siril-O) [inline HW/src/test/java/homework11/ArrayListTest.java]: why so you init user before each test ?

Comment (Siril-O) [reply HW/src/test/java/homework11/ArrayListTest.java]: does it changes between tests ?

File: HW/src/test/java/homework11/LinkedListTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import homework04.User;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class LinkedListTest {
```
Comment (Siril-O) [inline HW/src/test/java/homework11/LinkedListTest.java]: same issues as in  ArrayListTest

File: HW/src/test/java/homework11/ArrayListTest.java (hunk 2 above)
Comment (YeaaaaH) [reply HW/src/test/java/homework11/ArrayListTest.java]: Ah, agree with you, they are the same in each test

**Pull request merged**